okta-angular.js

```diff
@@ -0,0 +1,339 @@
+/* 	Author: Jordan Melberg */
+/** Copyright © 2016, Okta, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+
+/**
+ *	Custom Angular Wrapper for the Okta AuthSDK
+ *
+ *	To Use:
+ * 		Inject "OktaAuthClient" into your modules,
+ *		followed by "authClient" in your controllers, directives,
+ *		etc. if using a custom login form
+ *
+ *		Inject "widgetClient" in your controllers, directives,
+ *		etc if using the sign-in widget
+ */
+
+angular
+.module("OktaAuthClient", [])
+.factory("authClient", function($q) {
```

Comment:
What is the purpose of this authClient factory? I.e. why not just return an instance of the normal authClient? It seems like what we're doing is returning a new interface for existing authJs methods, with different ways to call known methods, and with different return values.